democracy/models/section.py

```diff
@@ -49,6 +49,8 @@ class Section(Commentable, StringIdBaseModel):
     content = models.TextField(verbose_name=_('content'), blank=True)
     plugin_identifier = models.CharField(verbose_name=_('plugin identifier'), blank=True, max_length=255)
     plugin_data = models.TextField(verbose_name=_('plugin data'), blank=True)
+    plugin_source = models.URLField(verbose_name=_('plugin source code'), blank=True)
```

Comment:
Are you sure these need to be per-section? The original idea was that they could be mapped via the `plugin identifier`s.

I.e.

```python
DEMOCRACY_PLUGINS = {
  "mapdon-hkr": "democracy.plugins.MapdonHkrPlugin",
}
```

```python
class MapdonHkrPlugin(Plugin):
    source = "https://example.com/js"
    fullscreen = True
```